packages/react-core/src/demos/examples/DashboardHeader.js

```diff
@@ -147,7 +146,7 @@ export default class DashboardHeader extends React.Component {
                 <Button aria-label="Help" variant={ButtonVariant.plain} icon={<QuestionCircleIcon />} />
               </ToolbarItem>
             </ToolbarGroup>
-            <ToolbarItem visibility={{ default: 'hidden', sm: 'visible', md: 'visible', lg: 'hidden' }}>
+            <ToolbarItem visibility={{ default: 'visible', sm: 'visible', md: 'visible', lg: 'hidden', xl: 'hidden' }}>
```

Comment:
Untested, but I believe all but `lg` are needed - it should be visible by default, and once it's set at a breakpoint, it persists through all breakpoints by default.

```suggestion
            <ToolbarItem visibility={{ lg: 'hidden' }}>
```